package.json

```diff
@@ -99,7 +102,11 @@
   "resolutions": {
     "@types/react": "18.2.28",
     "@types/react-dom": "18.2.13",
-    "postcss": "^8.3.11"
+    "postcss": "^8.3.11",
+    "string-width": "^4",
```

Comment:
Why do we need more of these resolutions? Are there some dependency incompatibilities?

Reply:
https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092